packages/functions_client/lib/src/types.dart

```diff
@@ -32,4 +33,8 @@ class FunctionException {
   final String? reasonPhrase;
 
   FunctionException({required this.status, this.details, this.reasonPhrase});
```

Comment:
This could be `const`.
```suggestion
  const FunctionException({required this.status, this.details, this.reasonPhrase});
```